travis/licenses.golden

```diff
@@ -15254,7 +15254,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ====================================================================================================
 LIBRARY: boringssl
 ORIGIN: ../../../third_party/boringssl/src/LICENSE
-TYPE: LicenseType.unknown
+TYPE: LicenseType.openssl
 FILE: ../../../third_party/boringssl/src/crypto/bn/asm/x86_64-gcc.c
 FILE: ../../../third_party/boringssl/src/crypto/err/asn1.errordata
 FILE: ../../../third_party/boringssl/src/crypto/err/bio.errordata
```

Comment:
Ah yep - that was me getting overly excited and pushing that second commit in that adds support for the new no-copyright block but, in my excitement, failing to update the golden file. Turns out the block in the second commit is in the existing version of that file. 

The upcoming Dart SDK roll adds a further OpenSSL block that I'll add handling for in the next PR.